airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -17,10 +17,13 @@
 """Executes task in a Kubernetes POD"""
 import re
 import warnings
+from functools import cached_property
```

Comment:
Airflow still supports Python 3.6 and 3.7, which do not have this function.

```suggestion
try:
    from functools import cached_property
except ImportError:
    from cached_property import cached_property
```

Reply:
thanks yeah i remember we have to handle this (and handle differently with providers) was just moving quickly and letting pycharm do the imports for me :)

Reply:
on that topic, can't wait to drop 3.7....  i like the walrus operator

Reply:
You need to wait till June 2023 I am afraid :(

Reply:
sigh...

Reply:
https://endoflife.date/python -> (23 Dec 2021)